lib/views/forget_password/forget_password_view_model.dart

```diff
@@ -6,11 +6,14 @@ import 'package:kayla_flutter_ic/views/forget_password/forget_password_state.dar
 
 class ForgetPasswordViewModel extends StateNotifier<ForgetPasswordState> {
   String? get emailWarningMessage => _emailWarningMessage;
+  String? get successMessage => _successMessage;
```

Comment:
you can add the `successMessage` into the `ForgetPasswordState.success`:
![image](https://user-images.githubusercontent.com/60863885/218697098-7b6b1171-da01-4ca5-802e-9bd8357a9365.png)